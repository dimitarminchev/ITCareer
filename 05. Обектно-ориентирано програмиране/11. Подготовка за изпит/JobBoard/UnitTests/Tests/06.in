AddCategory Restaurants
AddCategory It
AddJobOffer Restaurants Waiter Romance 1000 onsite Sofia
GetAverageSalary It
AddJobOffer It TechSupport MyTech 1500 remote True
AddJobOffer It OnSiteTechSupport MyTech 1700 onsite Burgas
AddJobOffer It Tester YourTest 2500 onsite Varna
GetAverageSalary It
GetOffersAboveSalary It 1600
End