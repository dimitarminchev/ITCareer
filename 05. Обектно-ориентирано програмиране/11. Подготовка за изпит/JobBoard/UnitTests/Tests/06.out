Created Category Restaurants!
Created Category It!
Created JobOffer Waiter in Category Restaurants!
The average salary is: 0.00 BGN
Created JobOffer TechSupport in Category It!
Created JobOffer OnSiteTechSupport in Category It!
Created JobOffer Tester in Category It!
The average salary is: 1900.00 BGN
Job Title: Tester
Company: YourTest
Salary: 2500.00 BGN
City: Varna
Job Title: OnSiteTechSupport
Company: MyTech
Salary: 1700.00 BGN
City: Burgas